context/app/static/js/components/workspaces/WorkspacesAuthenticated.jsx

```diff
@@ -5,30 +5,23 @@ import { AppContext } from 'js/components/Providers';
 import Description from 'js/shared-styles/sections/Description';
 import WorkspacesList from './WorkspacesList';
 
-function AuthenticatedWorkspacesDescription() {
+function WorkspacesAuthenticated() {
   const { workspacesToken } = useContext(AppContext);
 
-  const contact = (
-    <>
-      For any problems, notify Juan Puerto in <code>#workspaces</code>.
-    </>
-  );
-
   if (!workspacesToken) {
-    return (
-      <Description padding="20px">
-        The workspaces token request failed at login. {contact}{' '}
-        <OutboundIconLink href="https://github.com/hubmapconsortium/user_workspaces_server/issues/58">
-          Github issue
-        </OutboundIconLink>
-      </Description>
-    );
+    return <Description padding="20px">The workspaces token request failed at login.</Description>;
   }
 
   return (
     <>
       <Description padding="20px">
-        <strong>This is not yet ready for production use! {contact}</strong>{' '}
+        <strong>
```

Comment:
Again, I'd lean towards using `Typography` to keep things consistent... but good for now. Thanks!